task2/src/containers/MovieService.js

```diff
@@ -22,9 +21,24 @@ const genres = [
     'CRIME'
 ]
 
+const getMovies = () => (movies);
```

Comment:
to emulate api call => use Promise.resolve(movies)

the same for the others methods